SYCL/lit.cfg.py

```diff
@@ -313,6 +313,20 @@
 if find_executable('sycl-ls'):
     config.available_features.add('sycl-ls')
 
+# TODO properly set XPTIFW include and runtime dirs
+xptifw_lib = os.path.join(config.dpcpp_root_dir, 'lib')
+xptifw_dispatcher = ""
+if platform.system() == "Linux":
+    xptifw_dispatcher = os.path.join(xptifw_lib, 'libxptifw.so')
+elif platform.system() == "Windows":
+    xptifw_dispatcher = os.path.join(config.dpcpp_root_dir, 'bin', 'xptifw.dll')
+xptifw_includes = os.path.join(config.dpcpp_root_dir, 'include')
+if os.path.exists(xptifw_lib) and os.path.exists(os.path.join(xptifw_includes, 'xpti', 'xpti_trace_framework.h')):
+    config.available_features.add('xptifw')
+    config.substitutions.append(('%xptifw_lib', xptifw_lib))
+    config.substitutions.append(('%xptifw_dispatcher', xptifw_dispatcher))
+    config.substitutions.append(('%xptifw_includes', xptifw_includes))
```

Comment:
Do we really need separate substitutions? Are they used separately?
if not we can merge them